test/new-e2e/tests/sysprobe-functional/sysprobe_test.go

```diff
@@ -0,0 +1,128 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package sysprobefunctional
+
+import (
+	"flag"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	awshost "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/host"
+	"github.com/DataDog/datadog-agent/test/new-e2e/tests/windows"
+	windowsAgent "github.com/DataDog/datadog-agent/test/new-e2e/tests/windows/agent"
+	componentsos "github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
+	"github.com/stretchr/testify/require"
+)
+
+type vmSuite struct {
+	e2e.BaseSuite[environments.Host]
+}
+
+var (
+	kitchenDir string
+	testspath  string
+	reporoot   string
+	devMode    = flag.Bool("devmode", false, "run tests in dev mode")
+)
+
+func init() {
+	// Get the absolute path to the test assets directory
+	currDir, _ := os.Getwd()
+
+	reporoot, _ = filepath.Abs(filepath.Join(currDir, "..", "..", "..", ".."))
+	kitchenDir = filepath.Join(reporoot, "test", "kitchen", "site-cookbooks")
+	testspath = filepath.Join(kitchenDir, "dd-system-probe-check", "files", "default", "tests")
+}
+
+func TestVMSuite(t *testing.T) {
+	suiteParams := []e2e.SuiteOption{e2e.WithProvisioner(awshost.ProvisionerNoAgentNoFakeIntake(awshost.WithEC2InstanceOptions(ec2.WithOS(componentsos.WindowsDefault))))}
+	if *devMode {
+		suiteParams = append(suiteParams, e2e.WithDevMode())
+	}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
No need to explicit `devMode`, you can configure it in your local storage and the provider will automatically use your local settings https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/pkg/e2e/suite.go#L520

You can add it to your `~/.test_infra_config.yaml` under `configParams.devMode: true` or set the env variable `E2E_DEV_MODE=true`

Reply:
this was present in the sample I started with, and in fact is present in the startup example
https://github.com/DataDog/datadog-agent/blob/main/test/new-e2e/examples/vm_test.go#L29-L31

If this is not correct, I'd be happy to change but it's not clear whether it should actually be used or not.

Reply:
Ah, thank you for flagging this, I surely need to update the example !!